src/components/enterprise-user-subsidy/UserSubsidy.jsx

```diff
@@ -18,6 +19,8 @@ import { LOADING_SCREEN_READER_TEXT } from './data/constants';
 export const UserSubsidyContext = createContext();
 
 const UserSubsidy = ({ children }) => {
+  const user = getAuthenticatedUser();
+
   const { enterpriseConfig } = useContext(AppContext);
```

Comment:
nit: ok either way, but `authenticatedUser` is part of `AppContext` if you wanted to deconstruct it

Reply:
I'm curious, is there an easy way to query/figure out what is available in the contexts?

Reply:
Usually, you can look at the ContextProvider component or where it's used.

Reply:
@macdiesel TypeScript ;)  But with what we have today, as Long said, either look at the value passed to the ContextProvider or simply:

```
const contextValue = useContext(AppContext);
console.log(contextValue);
```